ee/clickhouse/migrations/0026_fix_materialized_window_and_session_ids.py

```diff
@@ -0,0 +1,41 @@
+from infi.clickhouse_orm import migrations
+
+from ee.clickhouse.materialized_columns.columns import get_materialized_columns, materialize
+from ee.clickhouse.materialized_columns.replication import clickhouse_is_replicated
+from posthog.client import sync_execute
+from posthog.settings import CLICKHOUSE_CLUSTER
+
+
+def rename_column_on_events_table(old_column_name, new_column_name):
+    if clickhouse_is_replicated():
+        sync_execute(
+            f"""
+                ALTER TABLE sharded_events
+                ON CLUSTER '{CLICKHOUSE_CLUSTER}'
+                RENAME COLUMN IF EXISTS {old_column_name} to {new_column_name}
+            """
+        )
+    sync_execute(
+        f"""
+            ALTER TABLE events
+            ON CLUSTER '{CLICKHOUSE_CLUSTER}'
+            RENAME COLUMN IF EXISTS {old_column_name} to {new_column_name}
+        """
+    )
+
+
+def create_materialized_columns(database):
+    properties = ["$session_id", "$window_id"]
+    for property_name in properties:
+        try:
+            materialize("events", property_name, property_name)
+        except ValueError:
```

Comment:
This seems like an implicit assumption that this will always be the exception raised for the desired case. Can we ensure we add tests for this case such that if the `materialize` raising characteristics change. One for where the property is already materialized and one where it isn't already materialized.